examples/fuse_examples/imaging/classification/cmmd/runner.py

```diff
@@ -260,17 +267,18 @@ def run_infer(paths : NDict , infer: NDict):
                                   shuffle=False, drop_last=False,
                                   collate_fn=CollateDefault(),
                                   num_workers=infer["num_workers"])
+    # load python lightning module
+    pl_module = LightningModuleDefault.load_from_checkpoint(checkpoint_file, model_dir=paths["model_dir"], model=model, map_location="cpu", strict=True)
+    # set the prediction keys to extract (the ones used be the evaluation function).
+    pl_module.set_predictions_keys(['model.output.classification', 'data.gt.probSevere']) # which keys to extract and dump into file
```

Comment:
those are stoic21 keys. Did you test it?

Reply:
fixed